wandb/__init__.py

```diff
@@ -40,6 +40,7 @@
 AlertLevel = wandb_sdk.AlertLevel
 Settings = wandb_sdk.Settings
 Config = wandb_sdk.Config
+automations = wandb_sdk.automations
```

Comment:
Why do this? Users can use `import wandb.automations` instead, which is more explicit and reduces the number of modules that `import wandb` imports.